subprojects/docs/src/snippets/java/toolchain-kotlin/groovy/build.gradle

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id "org.jetbrains.kotlin.jvm" version "1.4.10"
+}
+
+// tag::compiler-kotlin[]
+def compiler = javaToolchains.compilerFor {
+    languageVersion = JavaLanguageVersion.of(15)
```

Comment:
Just checking, is the kotlin compiler supported with java 15?

Reply:
Ah good point. Let me use 11 there.